x-pack/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -0,0 +1,248 @@
+      </EuiFormRow>
+
+      <EuiFormRow>
+        <EuiSwitch
+          label="Show legend"
+          checked={state.legend.isVisible}
+          data-test-subj="lnsXY_legendIsVisible"
+          onChange={() =>
+            setState({ ...state, legend: { ...state.legend, isVisible: !state.legend.isVisible } })
+          }
+        />
+      </EuiFormRow>
+
+      {state.legend.isVisible && (
+        <EuiFormRow label="Legend position">
+          <EuiButtonGroup
+            legend="Legend position"
+            data-test-subj="lnsXY_legendPosition"
+            name="legendPosition"
+            options={positionIcons}
+            idSelected={state.legend.position}
+            onChange={position =>
+              setState({ ...state, legend: { ...state.legend, position: position as Position } })
+            }
+            isIconOnly
+          />
+        </EuiFormRow>
+      )}
+
+      <EuiFormRow label="X Axis">
+        <>
+          <EuiFormRow label="Title">
+            <EuiFieldText
+              placeholder="Title"
+              data-test-subj="lnsXY_xTitle"
+              value={state.x.title}
+              onChange={e => setState({ ...state, x: { ...state.x, title: e.target.value } })}
+              aria-label="Title"
+            />
+          </EuiFormRow>
+
+          <EuiFormRow label="Value">
+            <NativeRenderer
+              data-test-subj="lnsXY_xDimensionPanel"
+              render={datasource.renderDimensionPanel}
+              nativeProps={{
+                columnId: state.x.accessor,
+                filterOperations: () => true,
+              }}
+            />
+          </EuiFormRow>
+
+          <EuiFormRow>
+            <EuiSwitch
+              label="Show gridlines"
+              data-test-subj="lnsXY_xShowGridlines"
+              checked={state.x.showGridlines}
+              onChange={() =>
+                setState({ ...state, x: { ...state.x, showGridlines: !state.x.showGridlines } })
+              }
+            />
+          </EuiFormRow>
+        </>
+      </EuiFormRow>
+
+      <EuiFormRow label="Y Axis">
+        <>
+          <EuiFormRow label="Title">
+            <EuiFieldText
+              placeholder="Title"
+              data-test-subj="lnsXY_yTitle"
+              value={state.y.title}
+              onChange={e => setState({ ...state, y: { ...state.y, title: e.target.value } })}
+              aria-label="Title"
+            />
+          </EuiFormRow>
+
+          <EuiFormRow label="Value">
+            <>
+              {state.y.accessors.map(accessor => (
+                <div key={accessor}>
+                  <NativeRenderer
+                    data-test-subj={`lnsXY_yDimensionPanel_${accessor}`}
+                    render={datasource.renderDimensionPanel}
+                    nativeProps={{
+                      columnId: accessor,
+                      filterOperations: (op: Operation) =>
+                        !op.isBucketed && op.dataType === 'number',
```

Comment:
We could also allow unbucketed dates here, right?

Reply:
I'm not sure... what would that do for a line / bar? It doesn't seem like something we'd want, but I may be missing something obvious?

Reply:
Example within the flights dataset: Average time of departure by airport. But the more I think about it the weirder it gets, we can add these edge cases later

Reply:
It's not really supported by elastic-charts yet, so it's actually safer to not allow bucketed columns on the Y axis.

Reply:
My comment was not about bucketed values on the Y axis but metrics based on dates. The current editor can do it:
<img width="321" alt="Screenshot 2019-06-04 at 08 43 33" src="https://user-images.githubusercontent.com/1508364/58857169-dd798d00-86a4-11e9-97ff-17181d6c04b5.png">

But I agree that we can add this later, better start with a smaller, but consistent and working set of features and expand step by step instead of pushing out a lot of half finished stuff.